src/idsk/components/banner/template.njk

```diff
@@ -0,0 +1,12 @@
+<div data-module="idsk-banner">
+    <div class="idsk-banner" role="contentinfo">
+        <div class="govuk-container-width">
+            <div class="idsk-banner__content {{ params.classes if params.classes }}">
+                <h2 class="govuk-heading-s">{{params.pageName}}</h2>
+                <h2 class="govuk-heading-m">
```

Comment:
hierarchia H-ciek tu pokulhava.. malo by to ist od najvacsieho dole.. cize bud `<h2>` a tak `<h3>` alebo `<h3>` a `<h4>`.. `<h1>` by som sa v tomto pripade vyhol kedze ta by mala byt len jeden krat na stranke a predpokladame ze na page-i este bude nejaky dolezitejsi nazov ako tento baner